cmd/lncli/commands.go

```diff
@@ -2446,8 +2446,13 @@ func verifyChanBackup(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	
+	if resp.String() == "" {
```

Comment:
Hmm, relying on the `String()` method here isn't a good idea IMO. Another option that would also benefit the RPC users would be to add a simple boolean to the `VerifyChanBackupResponse` in `lightning.proto`.
For example: `bool success = 1`, so the response would look like:
```
{
    "success": true
}
```

Reply:
Totally agree!